.github/workflows/kserve-test.yml

```diff
@@ -0,0 +1,118 @@
+name: "Kserve Smoke Test"
+
+on:
+  schedule:
+    - cron: "20 4 * * 1" # once a week
+  workflow_dispatch:
+
+  # push:
+  #   branches: [main]
+  #   paths:
+  #     - "!docs"
+  #     - "!demo"
+
+  # pull_request:
+
+env:
+  KIND_CLUSTER_NAME: "kserve-testing"
+  ISVC_NAME: "caikit-tgis-isvc"
+
+jobs:
+  smoke-test:
+    name: Kserve Smoke Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Free Disk Space
+        uses: jlumbroso/free-disk-space@main
+        with:
+          tool-cache: false
+
+          android: true
+          dotnet: true
+          haskell: true
+          large-packages: true
+          swap-storage: true
+
+      - name: Create k8s Kind Cluster
+        uses: helm/kind-action@v1.5.0
+        with:
+          cluster_name: ${{ env.KIND_CLUSTER_NAME }}
+
+      - name: Install kserve
+        run: |
+          # Use the hack/quick_install.sh script from the kserve repo
+          curl https://raw.githubusercontent.com/kserve/kserve/master/hack/quick_install.sh | bash
```

Comment:
This takes the latest version of KServe from upstream, we can consider to use odh fork and more importantly the latest version we include instead of the latest community: i.e. 0.12 will be soon released but ODH will remain on 0.11 for few more time

Reply:
Done